spec/features/scenarios/setting_up_a_school/second_academic_year_spec.rb

```diff
@@ -0,0 +1,307 @@
+                                  .return_to_manage_your_training
+      end
+
+      it_behaves_like "a school with an associated school cohort record"
+      it_behaves_like "a school with no induction programme record"
+      it_behaves_like "a school that cannot add a participant"
+    end
+  end
+
+  context "And they have chosen a full induction programme for the first academic year" do
+    let(:school_builder) do
+      school = NewSeeds::Scenarios::Schools::School.new
+                                                   .build
+                                                   .with_an_induction_tutor
+                                                   .with_partnership_in(cohort: Cohort.previous, lead_provider:, delivery_partner:)
+                                                   .chosen_fip_and_partnered_in(cohort: Cohort.previous)
+
+      PrivacyPolicy.current.accept! school.induction_tutor
+      school
+    end
+
+    context "after a core induction programme has been setup for the second academic year" do
+      before do
+        sign_in_as school.induction_tutor
+
+        Pages::SchoolDashboardPage.loaded
+                                  .report_changing_programme(programme_type: :change_to_cip)
+                                  .return_to_manage_your_training
+      end
+
+      it_behaves_like "a school with an associated school cohort record"
+      it_behaves_like "a school with a default core induction programme record"
+      it_behaves_like "a school that can add a participant"
+
+      context "and CIP materials has been setup for the second academic year" do
+        before do
+          Pages::SchoolDashboardPage.loaded
+                                    .add_cip_materials(core_induction_programme.name)
+        end
+
+        it_behaves_like "a school with a CIP materials record"
+        it_behaves_like "a school that can add a participant"
+      end
+
+      context "and an appropriate body has been setup for the second academic year" do
+        before do
+          Pages::SchoolDashboardPage.loaded
+                                    .add_appropriate_body(appropriate_body.name, appropriate_body.body_type)
+        end
+
+        it_behaves_like "a school with an appropriate body record"
+        it_behaves_like "a school that can add a participant"
+      end
+    end
+
+    context "after the same full induction programme has been setup for the second academic year" do
+      before do
+        sign_in_as school.induction_tutor
+
+        Pages::SchoolDashboardPage.loaded
+                                  .report_changing_programme(programme_type: :no_change)
+                                  .return_to_manage_your_training
+      end
+
+      it_behaves_like "a school with an associated school cohort record"
+      # TODO: Default induction programme is not copied to second academic year
+      it_behaves_like "a school with a default full induction programme record"
```

Comment:
When the SIT chooses to keep the same Inudction Programme for the second academic year a Default Induction Programme is not added to the second Cohort as expected.